.github/workflows/scheduled_workflow.yml

```diff
@@ -0,0 +1,38 @@
+name: ASPIRE Python Scheduled Workflow
+
+on:
+  workflow_dispatch:  # Manual "Run workflow" button
+  schedule:
+    - cron: '0 0 * * 0'  # Every Sunday at 00:00 UTC
+
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.10'
+    - name: Install dependencies
+      run: |
+        pip install tox
+    - name: Run Tox Check
+      run: tox -e check
+
+  scheduled-tests:
+    needs: check
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
```

Comment:
following up on our chat, maybe better to run on `develop` branch so we can potentially address the failure faster.

Reply:
Switched to only running on develop.

As mentioned above, I still need to test the workflow in a fork.